src/components/Repos.tsx

```diff
@@ -67,46 +73,71 @@ const Repos: React.FC = () => {
 
   return (
     <section className="h-full">
-      <div className="w-full py-5 pl-5 flex space-x-2 items-center">
-        <input
-          type="date"
-          className="bg-gray-800 hover:bg-gray-900 text-gray-200 font-bold py-3 px-4 rounded"
-          onChange={(e) => {
-            setDate(e.target.value);
-          }}
-          value={date}
-        />
-        <input
-          type="text"
-          placeholder="Search Repositories"
-          value={searchQuery}
-          onChange={(e) => setSearchQuery(e.target.value)}
-          className="bg-gray-800 hover:bg-gray-900 text-gray-200 font-bold py-3 px-4 rounded"
-        />
-        <Select
-          value={selectedOption}
-          onValueChange={(value) => {
-            setSelectedOption(value);
-          }}
+      <div className="flex flex-row items-center justify-between py-5 px-5">
+        <div className="w-full flex space-x-2 items-center">
+          <input
+            type="date"
+            className="bg-gray-800 hover:bg-gray-900 text-gray-200 font-bold py-3 px-4 rounded"
+            onChange={(e) => {
+              setDate(e.target.value);
+            }}
+            value={date}
+          />
+          <input
+            type="text"
+            placeholder="Search Repositories"
+            value={searchQuery}
+            onChange={(e) => setSearchQuery(e.target.value)}
+            className="bg-gray-800 hover:bg-gray-900 text-gray-200 font-bold py-3 px-4 rounded"
+          />
+          <Select
+            value={selectedOption}
+            onValueChange={(value) => {
+              setSelectedOption(value);
+            }}
+          >
+            <SelectTrigger className="w-[180px]">
+              <SelectValue placeholder="Daily" />
+            </SelectTrigger>
+            <SelectContent>
+              <SelectItem value="daily" defaultChecked={true}>
+                Daily
+              </SelectItem>
+              <SelectItem value="weekly">Weekly</SelectItem>
+              <SelectItem value="monthly">Monthly</SelectItem>
+            </SelectContent>
+          </Select>
+        </div>
+
+        <button
+          className="bg-gray-800 hover:bg-gray-900 text-gray-200 font-bold py-3 px-4 rounded text-nowrap"
+          onClick={() => setShowViewed(!showViewed)}
         >
-          <SelectTrigger className="w-[180px]">
-            <SelectValue placeholder="Daily" />
-          </SelectTrigger>
-          <SelectContent>
-            <SelectItem value="daily" defaultChecked={true}>
-              Daily
-            </SelectItem>
-            <SelectItem value="weekly">Weekly</SelectItem>
-            <SelectItem value="monthly">Monthly</SelectItem>
-          </SelectContent>
-        </Select>
+          {showViewed ? "Hide Viewed" : "Show Viewed"}
+        </button>
       </div>
 
+      <AnimatePresence>
```

Comment:
Make it in either way viewed

Reply:
Hey - just fixed the other comments, can you clarify what you mean on this one please?

Reply:
So currently, the button only shows when you click on the other button. I want the 2 buttons to be visible

Reply:
✅ Gotcha, updated that.